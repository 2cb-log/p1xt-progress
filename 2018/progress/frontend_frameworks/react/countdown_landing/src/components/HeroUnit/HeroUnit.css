.HeroUnit {
    text-align: center;
    display: flex;
    flex-direction: column;
    background: url(/assets/images/benjamin-voros-250200.jpg);
    background-size: 100vw;
    min-height: 500px;
    width: 100vw;
}

.HeroUnit__transparency {
    position: absolute;
    top: 0;
    left: 0;
    background: #272727;
    opacity: .8;
    height: 500px;
    width: 100vw;
}