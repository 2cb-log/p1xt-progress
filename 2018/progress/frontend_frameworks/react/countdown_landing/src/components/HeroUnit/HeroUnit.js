import React from 'react';
import './HeroUnit.css';

function HeroUnit(props) {
  return (
    <div className="HeroUnit">
      HeroUnit
    </div>
  );

}

export default HeroUnit;
