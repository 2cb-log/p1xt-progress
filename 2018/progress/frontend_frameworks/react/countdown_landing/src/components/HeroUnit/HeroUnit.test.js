import React from 'react';
import ReactDOM from 'react-dom';
import HeroUnit from './HeroUnit';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<HeroUnit />, div);
  ReactDOM.unmountComponentAtNode(div);
});
it('renders correctly', () => {
  const tree = renderer
    .create(<HeroUnit />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
