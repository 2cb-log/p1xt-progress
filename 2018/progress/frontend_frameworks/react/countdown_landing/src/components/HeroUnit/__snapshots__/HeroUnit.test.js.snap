// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroUnit /> renders correctly 1`] = `
<div
  className="HeroUnit"
>
  <div
    className="HeroUnit__transparency"
  />
  <button
    className="Logo"
  >
    Landing Page
  </button>
  <div
    className="Countdown"
  >
    Countdown
  </div>
  <div
    className="NotifyButton"
  >
    NotifyButton
  </div>
</div>
`;
